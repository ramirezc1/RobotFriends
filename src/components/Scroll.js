import React from "react";

const Scroll = ({ children }) => {
  return (
    <div
      style={{
        overflowY: "scroll",
        border: "5px solid black",
        height: "600px",
      }}
    >
      {children}
    </div>
  );
};

export default Scroll;
