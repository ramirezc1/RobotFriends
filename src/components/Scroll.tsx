import React from "react";
type Props={
  children?:JSX.Element
}
const Scroll = (props:Props) => {
  return (
    <div
      style={{
        overflowY: "scroll",
        border: "5px solid black",
        height: "600px",
      }}
    >
      {props.children}
    </div>
  );
};

export default Scroll;
