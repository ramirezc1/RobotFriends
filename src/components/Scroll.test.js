import { shallow } from "enzyme";
import Scroll from "./Scroll";

it("expect to render Scroll component", () =>
  expect(shallow(<Scroll></Scroll>)).toMatchSnapshot());
