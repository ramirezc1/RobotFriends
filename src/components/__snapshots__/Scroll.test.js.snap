// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Scroll component 1`] = `
<div
  style={
    Object {
      "border": "5px solid black",
      "height": "600px",
      "overflowY": "scroll",
    }
  }
/>
`;
